.github/files/build-all-projects.sh

```diff
@@ -51,42 +52,41 @@ for project in projects/packages/* projects/plugins/*; do
 	## clone, delete files in the clone, and copy (new) files over
 	# this handles file deletions, additions, and changes seamlessly
 
-	if jq -e '.scripts["build-production"]' composer.json &>/dev/null; then
-		echo "::group::Building ${GIT_SLUG}"
-
-		# If composer.json contains a reference to the monorepo repo, add one pointing to our production clones just before it.
-		# That allows us to pick up the built version for plugins like Jetpack.
-		# Also save the old contents to restore post-build to help with local testing.
-		OLDJSON=$(<composer.json)
-		JSON=$(jq --arg path "$BUILD_BASE/*/*" '( .repositories // [] | map( .options.monorepo or false ) | index(true) ) as $i | if $i != null then .repositories[$i:$i] |= [{ type: "path", url: $path, options: { monorepo: true } }] else . end' composer.json | "$BASE/tools/prettier" --parser=json-stringify)
-		if [[ "$JSON" != "$OLDJSON" ]]; then
-			echo "$JSON" > composer.json
-			if [[ -e "composer.lock" ]]; then
-				OLDLOCK=$(<composer.lock)
-				composer update --root-reqs --no-install
-			else
-				OLDLOCK=
-			fi
-		fi
-
-		if composer run-script --timeout=0 build-production; then
-			FAIL=false
+	echo "::group::Building ${GIT_SLUG}"
+
+	# If composer.json contains a reference to the monorepo repo, add one pointing to our production clones just before it.
+	# That allows us to pick up the built version for plugins like Jetpack.
+	# Also save the old contents to restore post-build to help with local testing.
+	OLDJSON=$(<composer.json)
+	JSON=$(jq --arg path "$BUILD_BASE/*/*" '( .repositories // [] | map( .options.monorepo or false ) | index(true) ) as $i | if $i != null then .repositories[$i:$i] |= [{ type: "path", url: $path, options: { monorepo: true } }] else . end' composer.json | "$BASE/tools/prettier" --parser=json-stringify)
+	if [[ "$JSON" != "$OLDJSON" ]]; then
+		echo "$JSON" > composer.json
+		if [[ -e "composer.lock" ]]; then
+			OLDLOCK=$(<composer.lock)
+			composer update --root-reqs --no-install
 		else
-			FAIL=true
+			OLDLOCK=
 		fi
+	fi
+	# Need to remove the "projects/" from the string since the CLI only looks for {type}/{project-name}.
+	SLUG=$(echo "$project" | cut -c 10-)
+	if jetpack build "${SLUG}"; then
```

Comment:
Something is wrong here, it's failing to build all the packages.

Also you need to pass `--production`, otherwise it does a dev build, which symlinks the vendor packages, which fails to copy.

Reply:
🤦 I told myself to do that too. eba48f475 for the `--production`.

Reply:
Fixed via 9b67caa . The CLI assumed there would be a `scripts` object in the composer.json, which didn't seem to matter locally, but failed here. Added a more explicit check.